docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -115,9 +115,8 @@ something provider-specific. The tabs below will contain a notice about what fla
 on `kubeadm init` are required.
 - Unless otherwise specified, kubeadm uses the default gateway's network interface 
 to advertise the master's IP. If you want to use a different network interface, specify 
-`--apiserver-advertise-address=<ip-address>` argument to `kubeadm init`.
-- If you would like to customise control plane components, you can do so by providing 
-extra args to each one, as documented [here](/docs/admin/kubeadm#custom-args).
+`--apiserver-advertise-address=<ip-address>` argument to `kubeadm init`. To deploy an IPv6 Kubernetes cluster using IPv6 addressing, you must specify an IPv6 `--apiserver-advertise-address=<IPv6-address>` 
```

Comment:
Please update this portion:

you must specify an IPv6 address `--apiserver-advertise-address=<ipv6-address>`